dataland-frontend/src/components/general/SelectReportingPeriodDialog.vue

```diff
@@ -2,9 +2,14 @@
   <div data-test="select-reporting-period-dialog">
     <h4 class="title">SELECT YEAR</h4>
     <div class="three-in-row" data-test="reporting-periods">
-      <router-link v-for="(el, index) in dataTableContents" :key="index" class="link" :to="el.editUrl">{{
-        el.reportingPeriod
-      }}</router-link>
+      <a
+        v-for="(el, index) in dataTableContents"
```

Comment:
"element" instead of "el" would be a little more readable

Reply:
done.